internal/db/leveldb.go

```diff
@@ -50,9 +50,12 @@ func clock(v uint64) uint64 {
 }
 
 const (
-	keyTypeDevice = iota
-	keyTypeGlobal
-	keyTypeBlock
+	KeyTypeDevice = iota
+	KeyTypeGlobal
+	KeyTypeBlock
+	KeyTypeDeviceStatistic
+	KeyTypeFolderStatistic
```

Comment:
Yes, but the format of the saved stats is changed anyhow due to how NamespacedKV works, so I don't think this matters...